snuba/datasets/storages/metrics.py

```diff
@@ -0,0 +1,78 @@
+from snuba.datasets.metrics_processor import MetricsProcessor
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Array,
+    Column,
+    ColumnSet,
+    DateTime,
+    Float,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas.tables import TableSchema, WritableTableSchema
+from snuba.datasets.storage import ReadableTableStorage, WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.table_storage import build_kafka_stream_loader_from_settings
+
+buckets_columns = ColumnSet(
+    [
+        Column("org_id", UInt(64)),
+        Column("project_id", UInt(64)),
+        Column("metric_id", UInt(64)),
+        Column("metric_type", String()),
+        Column("timestamp", DateTime()),
+        Column("tags", Nested([Column("key", UInt(64)), Column("value", UInt(64))])),
+        Column("sum_values", Float(64)),
```

Comment:
Should these be added later, in the same PR that adds support for these types?

Also, why is this column "sum_values" but it only contains a single number?

Reply:
Actually that is a good point. I decided to break the schema in to multiple after creating this table. Removed